plugin/nvimtree.lua

```diff
@@ -0,0 +1,6 @@
+-- Nvim Tree
+vim.g.loaded_netrw = 1 -- required
+vim.g.loaded_netrwPlugin = 1 -- required
+vim.opt.termguicolors = true -- required
```

Comment:
some of these are already in settings.lua and could have put the rest there

Reply:
ok...